charts/liquibase-migration/templates/_secretDefinition.yaml

```diff
@@ -0,0 +1,28 @@
+{{- define "liquibase-migration.secretDefinition" -}}
+{{- if .Values.migration.enabled -}}
+{{- $dbUsernameKey := .Values.vault.migration.dbUsernameKey | required "A valid vault.migration.dbUsernameKey is required" -}}
+{{- $dbPasswordKey := .Values.vault.migration.dbPasswordKey | required "A valid vault.migration.dbPasswordKey is required" -}}
+{{- $vaultPath := .Values.vault.migration.path | required "A valid vault.migration.path is required" -}}
+{{- $appName := .Values.migration.appName | default .Release.Name -}}
+{{- $labelRef := .Values.migration.labelRef | default (printf "%s.labels" $appName) -}}
+{{- $syncWave := .Values.migration.syncWave | default "-1" -}}
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: {{ $appName }}-migration-secretdef
+  labels: {{- include $labelRef . | nindent 4 }}
+  annotations:
+    argocd.argoproj.io/sync-wave: {{ $syncWave | quote }}
+spec:
+  name: {{ $appName }}-sql-secrets
```

Comment:
could you include `migration` in the name of the secret here too just to avoid potential confusion.